grid_problem.py

```diff
@@ -29,16 +39,122 @@ def get_cell_borders(i):
 
 class GridProblem:
     def __init__(self):
-        #populations = [int(random.random()*100 + 1) for i in range(w*h)]
+        # self.populations = [int(random.random()*100 + 1) for i in range(W * H)]
         self.populations = [1 for i in range(W*H)]
         self.areas = [1 for i in range(W*H)]
         self.borders = [get_cell_borders(i) for i in range(W*H)]
         self.num_districts = NUM_DISTRICTS
         self.total_population = reduce(lambda x, y: x + y, self.populations)
 
+    def split_district(self, district):
```

Comment:
In the rest of the code, a "district" is defined as an array of sets of ids.  Since its different here it might be confusing.

Though honestly you can just merge this thing and it'll be fine since I doubt anybody else is gonna touch this code.